packages/components/src/components/Forms/FormText.tsx

```diff
@@ -0,0 +1,167 @@
+import { Text } from 'react-native'
+import React, { FC } from 'react'
+
+import { StringOrTextWithA11y } from '../../types/common'
+import { getA11yText, getDisplayText } from '../../utils'
+import { useTheme } from '../../utils'
+import { useTranslation } from 'react-i18next'
+
+export type FormTextProps = {
+  text: StringOrTextWithA11y
+}
+
+/**
+ * Fonts (TODO: replace with typography tokens)
+ */
+const fontRegular = 'SourceSansPro-Regular'
+const fontBold = 'SourceSansPro-Bold'
+
+const fontHeader = {
+  fontFamily: fontRegular,
+  fontSize: 20,
+  lineHeight: 30,
+}
+
+const fontHint = {
+  fontFamily: fontRegular,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+const fontError = {
+  fontFamily: fontBold,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+export const fontLabel = {
+  fontSize: 20,
+  lineHeight: 30,
+}
+
+const fontDescription = {
+  fontFamily: fontRegular,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+/**
+ * Header
+ */
+export type HeaderProps = FormTextProps & {
+  required?: boolean
+}
+
+export const Header: FC<HeaderProps> = ({ text, required }) => {
+  const theme = useTheme()
+  const { t } = useTranslation()
+
+  const textStyle = {
+    ...fontHeader,
+    color: theme.vadsColorForegroundDefault,
+  }
+
+  const requiredStyle = {
+    color: theme.vadsColorForegroundError,
+  }
+
+  const ariaLabel = required
+    ? getA11yText(text) + ', ' + t('required')
+    : getA11yText(text)
+
+  return (
+    <Text aria-label={ariaLabel} style={textStyle}>
+      {getDisplayText(text)}
+      {required && <Text style={requiredStyle}>{` (*${t('required')})`}</Text>}
+    </Text>
+  )
+}
+
+/**
+ * Hint
```

Comment:
I'm willing to let it slide as these are fairly closely tied to forms generally and not very multifunctional, but am noting that I'm not sure they strictly pass the `New functions have proper TSDoc annotations` checklist item.

Reply:
Updated with more descriptive comments and comments above prop types